### ISSUE TITLE
Add support for installation via PhiVE

### ISSUE THREAD

**Issue by amenk** [consolidation/Robo#667]
Either by using github releases or by supplying a phive.xml + gpg signatures.

See https://phar.io/

**Comment by andrzejkupczyk** [comment]
This is currently possible using the `--force-accept-unsigned` option but it should not be necessary.

https://phar.io/distribute-your-own.html
https://github.com/sebastianbergmann/phar-site-generator

**Comment by prudloff-insite** [comment]
I agree this would be useful, it would allow us to easily install different versions of Robo on our CI for our different projects.

**Comment by greg-1-anderson** [comment]
A PR would be welcome. I'll hook up the CI.
